### DISCUSSION TITLE
127a2f62-7746-4628-aed2-7246c610597a

### DISCUSSION THREAD

**Discussion by giscus** [varofla/blog_giscus#25] [General]
# 127a2f62-7746-4628-aed2-7246c610597a

Beepy Likely 프로젝트에 사용할 브레이크아웃 보드를 제작했습니다.

https://www.varofla.com/127a2f62-7746-4628-aed2-7246c610597a

**Comment by year1984** [comment]
안녕하세요, 이 프로젝트가 어찌 진행이 되고 있나요? 졸업한 후에 한참동안 임베디드 개발을 잊고 있다가, 최근에 취미로 깨작거리는 도중입니다. beepberry를 발견하고는 차완기님과 같은 생각에 도달했는데, 저 키보드 케이블 문제 때문에 막혔었습니다. 작업 내용이 상당히 흥미롭습니다. 이후 진행이 어찌되어 가고 있나요?

**Reply by varofla** [reply]
안녕하세요, 우선 흥미롭게 봐주셔서 감사합니다.

사실 몇 달 전에 이미 키보드 입력 기능은 완성된 상태입니다.
학업을 병행하는 직장의 노예(...)이다 보니 개인적인 일정이 또한 겹쳐 포스팅을 잠시 미뤄두고 있었는데요,
요즈음 여유가 다시 생기고 있어 이후 포스팅을 준비하고 있었습니다. (그러나 학기말이 다가오고 있습니다 :sweat_smile:)
키보드 드라이버 구현, 관련 RTOS Task 처리에 대한 부분을 우선 다룬 이후, PCB 설계 파트로 넘어가지 않을까 싶습니다.
[테스트 영상](https://youtu.be/IfYy4Sb9S4M?si=EIMFpTJgt0Vw1Dy8)이 얼른 공개될 날이 왔으면 좋겠습니다.

키보드 케이블은 포스팅 본문에 언급한 내용처럼 `BM14B(0.8)-24DS-0.4V`를 사용하면 딱입니다. 핀아웃 역시 [제가 제작한 라이브러리](https://github.com/varofla/BM14B-eagle-pinout?tab=readme-ov-file)대로 해주시면 딱 맞습니다.

언제가 될 지 잘 모르겠지만, 다음 포스팅에서 뵙겠습니다 :)
